### ISSUE TITLE
And what about the rest of us??

### ISSUE THREAD

**Issue by oywino** [sabeechen/hassio-google-drive-backup#1084]
When will you upgrade this fine integration to work on non-supervised HA installation, like those of us running HA Core in a docker container?

**Comment by hax4dazy** [comment]
I got annoyed with this as well and kinda just added the container to my docker-compose file but that resulted in an error which I'm not sure how to get out of. I guess this just isn't made to be ran stand-alone and needs HA to be installed in a supervised way. Pretty sad that the rest of us don't get something cool like this, oh well.

**Comment by freddeh** [comment]
+1 :(

**Comment by jeffreyvangorkum** [comment]
Yeah, would definitely love a way to run this without a supervised install.
